chat/src/main/java/io/skygear/plugins/chat/ChatContainer.java

```diff
@@ -659,9 +661,10 @@ public void onSucc(@Nullable List<Message> messages) {
                                 mMap.put(m.getId(), m);
                             }
                             for (UserConversation uc: userConversations) {
-                               if (uc.getLastReadMessageId() != null) {
-                                   uc.lastMessage = mMap.get(uc.getLastReadMessageId());
-                               }
+                                Conversation c = uc.conversation;
+                                if (c.getLastMessageId() != null) {
+                                    uc.lastMessage = mMap.get(c.getLastMessageId());
+                                }
```

Comment:
@ben181231 let skip last _read_ message transient include for now. Seem it is not worth to add extra param for it. Seem none is requesting and no use case.